src/coreclr/vm/i386/stublinkerx86.cpp

```diff
@@ -3060,6 +3060,8 @@ VOID StubLinkerCPU::EmitComputedInstantiatingMethodStub(MethodDesc* pSharedMD, s
 #ifdef TARGET_AMD64
 VOID StubLinkerCPU::EmitLoadMethodAddressIntoAX(MethodDesc *pMD)
 {
+    pMD->EnsureSlotFilled();
```

Comment:
All these should have comments. I'll stop commenting.

Reply:
Instead of comments, it may be better to introduce a method that fill the slot and returns its value or address, and call them instead of `GetStableEntryPoint` or `GetAddrOfSlot` below. It would make it self-describing.

Reply:
It turns out we already had that function, its TryGetMulticallableAddrOfCode. The behavior is almost what we want, so I've added a new flag, and now this path generates very slightly more optimal code. (TryGetMulticallableAddrOfCode has a slight optimization around handling of Tier1 code.)